pelican/plugins/webassets/webassets.py

```diff
@@ -47,33 +47,67 @@ def create_assets_env(generator):
     assets_destination = os.path.join(generator.output_path, theme_static_dir)
     generator.env.assets_environment = Environment(assets_destination, theme_static_dir)
 
-    if "ASSET_CONFIG" in generator.settings:
-        for item in generator.settings["ASSET_CONFIG"]:
-            generator.env.assets_environment.config[item[0]] = item[1]
+    # TODO: remove deprcated variables in 2022
```

Comment:
Typo: `deprcated` -> `deprecated`.

Reply:
Haha one day I'll learn how to spell that word or get better at [flyspell](https://www.emacswiki.org/emacs/FlySpell)

I've amended this to the 424c27c **- deprecate ASSET_CONFIG in favor of WEBASSETS_CONFIG** commit